.gitignore

```diff
@@ -3,7 +3,7 @@ dist
 node_modules
 .vscode-test/
 *.vsix
-package-lock.json
+/package-lock.json
```

Comment:
Do we have multiple package lock files or why is this changed?

Reply:
Yeah there's one in cursorless-nx subdir now.  We'll merge things when we migrate to nx but for now just wanted to have a lock file in there